index.bs

```diff
@@ -1903,10 +1903,10 @@ for vertical cues, it's the horizontal offset. The offset is for the <a lt="WebV
 alignment">start</a>, <a lt="WebVTT cue line center alignment">center</a>, or <a lt="WebVTT cue line
 end alignment">end</a> of the cue box, depending on the <a>WebVTT cue line alignment</a> value - <a
 lt="WebVTT cue line start alignment">start</a> by default. The offset can be given either as a
-percentage of the video dimension or as a line number. Line numbers are based on the size of the
-first line of the cue. Positive line numbers count from the start of the video viewport (the first
-line is numbered 0), negative line numbers from the end of the viewport (the last line is numbered
-&#x2212;1).</p>
+percentage of the relevant writing-mode dependent video dimension or as a line number. Line numbers
```

Comment:
The video dimension not the video viewport dimension?

Reply:
The "video viewport" is the rendering area of the video, which is the only area that we're able to render anything onto, so for all intents and purposes it's really just a description of the video dimensions.

Reply:
@nigelmegitt I can use "video viewport dimension" if you think it makes more sense.

Reply:
@silviapfeiffer that would seem more consistent, modulo the comments on the related issue, that viewport units are not video viewport units.

Reply:
@nigelmegitt for now, I've made sure to use "video viewport" everywhere to clarify for this issue. I'll address the definition of the term "video viewport" in #375 .

Reply:
OK, thanks @silviapfeiffer